micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmGcMetrics.java

```diff
@@ -258,7 +259,12 @@ public void close() {
             put("MarkSweepCompact", OLD);
             put("PS MarkSweep", OLD);
             put("PS Scavenge", YOUNG);
-            put("ParNew", YOUNG);
+            put("ParNew", YOUNG); 
+            put("global", OLD);
+            put("scavenge", YOUNG);
+            put("partial gc", YOUNG);
+            put("global garbage collect", OLD);
+            put("Epsilon", OLD);
```

Comment:
I had added it for completeness but you're completely right it won't issue any notifications.  Will remove

Reply:
I did notice some notifications for it when I was testing, but I'm not sure when they happen since they should be a no-op. Somehow it still reports some time taken for it's "GC event". We could consider supporting it if we understand what it means and there's interest from users.